domains/rogo.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "owner": "BenRogozinski",
+    "discord": "BenDaBuilder",
+    "email": "brogodev@gmail.com"
+  },
+  "record": {
+    "A": "5.161.233.190",
+    "AAAA": "2a01:4ff:f0:dd7c::1"
```

```suggestion
    "AAAA": ["2a01:4ff:f0:dd7c::1"]
```
this too must be in an array